cypress/e2e/news.cy.js

```diff
@@ -0,0 +1,24 @@
+describe("Work & Co News links test", () => {
+  it("Opens W&C site/News, iterates and opens each link, checks H1 for each page", () => {
+    cy.visit("https://work.co/");
```

Comment:
Some of your older tests are now failing because youhave changed baseURL. 

There are multiple ways around this. 

One of them is to ignore baseconfig and always use .visit in the tests. 
Another one is to create separate config files for each of tests. You can have multiple config files in the repo. This is good for real complex projects. 

YOu have not made a mistake here this is just info.

Reply:
Thanks, I'll try to create separate config files for saucedemo and W&C tests